src/pages/document/[id].tsx

```diff
@@ -283,6 +298,32 @@ const FamilyPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({ pa
               )}
             </section>
           ))}
+
+          <section className="mt-10" data-cy="top-documents">
+            <h3 className="mb-4">Documents</h3>
```

Comment:
I think this title needs to be slightly more descriptive, "Search for more", "Search in {countryName}" perhaps.

Might be worth speaking to @alaniswright on this

Reply:
I think this component just needs to include what is in the blue box.

We don't need the title "Documents" that appears on the country page to also appear when we repeat this element at the bottom of document pages

Reply:
Sorted.